dct_backend/dct.py

```diff
@@ -1,17 +1,29 @@
 from food_item import FoodItem
+from station import Station 
 
 class DCT():
     def __init__(self):
         self.menu = []
-        self.stations = []
+        self.stations = {}
```

Comment:
@oreorz  is "station" your dictionary key or food item?

Reply:
So the name of the station is the key and it will hold it's corresponding station item. In the station item, there is also a name variable though